docs/source/noise/column_noise.rst

```diff
@@ -19,6 +19,30 @@ Misreport age
 Write the wrong digits
 ----------------------
 
+Sometimes people may write the wrong number for numeric data such as a street
```

Comment:
It's interesting to describe this as numeric data -- it isn't, from a programming perspective, but maybe this is how our users will think about this?

Reply:
Maybe I could say "fields that include numbers" or something?

Reply:
Sounds good to me!